pkg/sql/tablewriter.go

```diff
@@ -227,13 +220,6 @@ func (tu *tableUpserter) walkExprs(walk func(desc string, index int, expr parser
 	}
 }
 
-func (tu *tableUpserter) start() error {
-	if tu.evaler != nil {
-		return tu.evaler.start()
```

Comment:
The start() machinery for the evaler had no other purpose than starting the sub-query plans. Now this is done by startPlans() by asking walkPlan() to visit all sub-query plans, and that will automatically grab the evaler's sub-queries via its (other) walkExprs() method.